prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/site-overview-page/site-overview-page.component.ts

```diff
@@ -47,16 +66,56 @@ export class SiteOverviewPageComponent implements OnInit {
     }
   }
 
+  public saveVendor(): void {
+    const vendor = this.vendors.value;
+    const existingVendor = this.healthAuthorityVendors.find((vendor: VendorConfig) => vendor.code === this.site.healthAuthorityVendor[0].vendorCode).name;
```

Comment:
Getting a runtime error on this line of code 

![image](https://github.com/bcgov/moh-prime/assets/7586665/4df64f6d-bdc0-4066-bd8c-9bff551d831e)

Reply:
this is fixed.